src/appendices/entity-table.md

```diff
@@ -11,7 +11,7 @@ Entity formats indicate whether the value is alphanumeric
 
 A general introduction to entities is given in the section on
 [filename structure](../02-common-principles.md#file-name-structure),
-while entity definitions are in [Appendix IX](09-entities.md).
+while entity definitions are in [Appendix IX](entities.md).
```

Comment:
Appendix IX no longer seems correct. What about:

```suggestion
while entity definitions are in [Entities](entities.md).
```

Reply:
I tackled this in a recent commit. I basically went with `the [{Name} Appendix]` in most cases.